tests/fixtures-valid.js

```diff
@@ -27,21 +60,35 @@ const uniqueValues = {
 };
 
 const promises = [];
-
-// search fixture files
 const fixturePath = path.join(__dirname, `..`, `fixtures`);
-for (const manKey of fs.readdirSync(fixturePath)) {
-  const manDir = path.join(fixturePath, manKey);
-
-  // files in manufacturer directory
-  if (fs.statSync(manDir).isDirectory()) {
-    for (const file of fs.readdirSync(manDir)) {
-      if (path.extname(file) === `.json`) {
-        const fixKey = path.basename(file, `.json`);
-        handleFixtureFile(manKey, fixKey);
+
+if (args.a) {
+  for (const manKey of fs.readdirSync(fixturePath)) {
+    const manDir = path.join(fixturePath, manKey);
+
+    // files in manufacturer directory
+    if (fs.statSync(manDir).isDirectory()) {
+      for (const file of fs.readdirSync(manDir)) {
+        if (path.extname(file) === `.json`) {
+          const fixKey = path.basename(file, `.json`);
+          handleFixtureFile(manKey, fixKey);
+        }
       }
     }
   }
+  checkManufacturers();
+}
+else {
+  // sanitize given path
+  fixturePaths = fixturePaths.map(relativePath => path.resolve(relativePath));
+  for (const fixPath of fixturePaths) {
+    if (path.extname(fixPath) !== `.json`) {
```

Comment:
Could we either output a warning here to inform the user about the skipped file, or even allow the shorthand `manKey/fixKey` syntax? If it's not a file at all, the error will throw when the file can't be read.

Reply:
It would make me more comfortable to do that when the user chooses to (by setting a verbose flag or similar), but I don't want to introduce that complexity.

Reply:
Okay, makes sense. Could you please output a console warning then?

Reply:
that was just a comment, I'll still implement both for now.

Reply:
Ooops, sorry I read over *verbose* in your comment, so mine doesn't make sense as an answer :sweat_smile:

I don't think this warning should only be shown when the user chooses verbose output, since it's likely that they called the script with wrong arguments and might not understand why the fixture is not validated.

Reply:
Yeah, but when that is the case, he can set the verbose flag for troubleshooting purposes like that. In case we ever do create solution where the fixture files can be validated individually, a user just might specify a `directory/*` where also many other files could be that aren't fixture definitions and then his entire output would be full of (in that case) irrelevant warnings.

Reply:
nvm regarding the fixKey/manKey shortcut. I would be more comfortable with that after I refactored the file (which would make more sense in a seperate PR).